### ISSUE TITLE
Load image with specific size

### ISSUE THREAD

**Issue by youngminpark2559** [xournalpp/xournalpp#4183]
**Is your feature request related to a problem? Please describe.**
When I load large size images, I should always resize them into small size

**Describe the solution you'd like**
- Memorizing the size of image which was last loaded, and when image is loaded newly, this image should be loaded with the memorized size
- Or, it would be great if I could specify certain area or region. After that I use ctrl+shift+i or drag-and-drop to load image
  Loaded image onto that region should have the specific designated size

**Comment by dov** [comment]
I agree with this. I was just about to open a feature request about something very similar.

Here is my suggested change:

- When in image mode and clicking on the canvas you will **not** get a file select dialog immediately. 
- The mouse click should starts the dragging of an on canvas rectangle. 
- On mouse release When the file select dialog is opened. 
- After selecting the image, it will be rescaled to fit into the dragged rectangle previously chosen and then put on the canvas.

**Comment by ysngshn** [comment]
At least it would be nice to have a **consistent** image insert size based on image resolution and pdf page size/DPI, etc. I am trying to insert the same signature saved as png and depending on where I click, it is inserted in a different size.

**Comment by p0mm** [comment]
I really like the idea with specifying the area for the image before selecting the image -> I would like to implement this!
I also agree that image insert size should be consistent, i have played around with inserting images:

https://github.com/xournalpp/xournalpp/assets/125290879/01cb191c-d737-43ca-9ce5-f70619fdfa8b

While there is reasons for the way xournalpp currently does it (zoom level & how close to the edge one adds an image) it doesn't feel very intuitive and i hope to improve it:
- maybe account for zoom? most users probably aren't working with 100% zoom, so pictures added will always be larger / smaller when they are added with original size for 100% zoom -> this could be changed to original size at current zoom?  
- allow user to grab and move image directly after inserting it (currently you add the same image again? which feels buggy)
- don't make the image smaller just because it was added closer to the edge, let the user decide to move or resize the image

I hope to open a PR soon, please feel welcome to suggest alternative behavior if you have any ideas - I'd be willing to try and implement them as well, so we can choose the best behavior in the end :)
